partner_event/models/event_registration.py

```diff
@@ -64,7 +64,13 @@ def _onchange_partner_id(self):
             get_attendee_partner_address = {
                 "get_attendee_partner_address": self.attendee_partner_id,
             }
-            return super(
-                EventRegistration, self.with_context(**get_attendee_partner_address)
-            )._onchange_partner_id()
-        return super(EventRegistration, self)._onchange_partner_id()
+            # onchange for partner_id removed in v16 core - including functionality here
```

Comment:
Why not changing the whole logic to a compute readonly=False store=True as it's donde with the other fields? (e. g: https://github.com/OCA/OCB/blob/4860a93e28a82003f881e5c21ddc2dd4229e14a8/addons/event/models/event_registration.py#L62-L69

Reply:
I'm sorry I don't understand the question. Are you able to expand any further? Do you mean: change the definition of the attendee_partner_id field to be more like the definition of the email field in the core code? E.g.:

`email = fields.Char(string='Email', compute='_compute_email', readonly=False, store=True, tracking=11)
`